### DISCUSSION TITLE
Two-Factor Authentication for password reset

### DISCUSSION THREAD

**Discussion by MichaelFBA** [directus/directus#21939] [Feature Requests]
### Summary

We had a security researcher reach out to let us know when Two-Factor Authentication is enabled for an account, the reset password functionality doesnt require the one time code. It would be great if this feature could be added

### Motivation

Security

### Detailed Design

We had a security researcher reach out to let us know when Two-Factor Authentication is enabled for an account, the reset password functionality doesnt require the one time code. It would be great if this feature could be added

### Must Have:
- Similar to login, but implement two factor auth on reset password

### Drawbacks

Implementation code

### Alternatives

webauth

### Adoption Strategy

It should be transparent for others

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
